tools/astarte_export/lib/astarte/fetchdata/fetchdata.ex

```diff
@@ -3,6 +3,15 @@ defmodule Astarte.Export.FetchData do
   alias Astarte.Core.CQLUtils
   alias Astarte.Export.FetchData.Queries
 
+  @native_types [
+    :booleanarray,
+    :datetimearray,
```

Comment:
Be careful here, datetime data should be converted to iso 8601 format (see lines 309-311)